int8 type
int8 default_point
RotationPose camera_rotation_pose

